src/components/checkbox/examples/checkbox-readonly.tsx

```diff
@@ -45,15 +44,21 @@ export class CheckboxReadonlyExample {
             <limel-checkbox
                 disabled={this.disabled}
                 label="Subscribe to email newsletters"
-                readonlyProps={{
-                    trueIcon: 'news',
-                    falseIcon: {
-                        name: 'cancel_subscription',
-                        color: 'rgb(var(--color-orange-default))',
+                readonlyLabels={[
+                    {
+                        value: true,
+                        icon: 'news',
+                        text: 'Is subscribed to receive newsletters',
```

Comment:
🤔 these don't seem to work for `true`

Reply:
cc42437a8c21ff9e60fa86f940011d4ce5f90d81